src/types/index.ts

```diff
@@ -68,12 +71,15 @@ interface CreateGroundPlacesParams {
   name: string;
   longitude: number;
   latitude: number;
+  address?: string;
 }
 
 interface UpdateGroundPlaceProperties {
+  [key: string]: number | string;
```

Comment:
idem, what is thee use of this ? it removes a lot of strictness in the typing

Reply:
Same answer as above